draft-packages/menu-list/KaizenDraft/MenuList/MenuList.scss

```diff
@@ -31,13 +31,13 @@ $side-padding: 3/4 * $ca-grid;
 }
 
 .header__title {
-  @include ca-type-ideal-labels-and-legends;
+  @include ca-type-inter-labels-and-legends;
```

Comment:
Can you change this to H6? Uppercase text should be completely stamped out in Kaizen: https://www.figma.com/file/3K0N4Pp0Dtyo37uAASamJI/Mapping-of-Typographic-Styles?node-id=0%3A1